tesseract_motion_planners/ompl/src/ompl_motion_planner.cpp

```diff
@@ -104,439 +95,237 @@ bool OMPLMotionPlanner::terminate()
   return false;
 }
 
+void OMPLMotionPlanner::clear() {}
+
 tesseract_common::StatusCode OMPLMotionPlanner::solve(const PlannerRequest& request,
                                                       PlannerResponse& response,
                                                       bool verbose) const
 {
+  auto status_category_ = std::make_shared<const OMPLMotionPlannerStatusCategory>(name_);
+
+  // Check the format of the request
   if (!checkUserInput(request))  // NOLINT
   {
     response.status =
         tesseract_common::StatusCode(OMPLMotionPlannerStatusCategory::ErrorInvalidInput, status_category_);
     return response.status;
   }
-  std::vector<OMPLProblem::Ptr> problem;
-  if (request.data)
-  {
-    problem = *std::static_pointer_cast<std::vector<OMPLProblem::Ptr>>(request.data);
-  }
-  else
-  {
-    try
-    {
-      problem = createProblems(request);
-    }
-    catch (std::exception& e)
-    {
-      CONSOLE_BRIDGE_logError("OMPLPlanner failed to generate problem: %s.", e.what());
-      response.status =
-          tesseract_common::StatusCode(OMPLMotionPlannerStatusCategory::ErrorInvalidInput, status_category_);
-      return response.status;
-    }
-
-    response.data = std::make_shared<std::vector<OMPLProblem::Ptr>>(problem);
-  }
 
   // If the verbose set the log level to debug.
   if (verbose)
     console_bridge::setLogLevel(console_bridge::LogLevel::CONSOLE_BRIDGE_LOG_DEBUG);
 
-  /// @todo: Need to expand this to support multiple motion plans leveraging taskflow
-  for (auto& p : problem)
+  // Get the planner profile
+  OMPLPlannerParameters params;
+  try
   {
-    auto parallel_plan = std::make_shared<ompl::tools::ParallelPlan>(p->simple_setup->getProblemDefinition());
+    const std::string profile_name =
+        getProfileString(name_, request.instructions.getProfile(), request.plan_profile_remapping);
```

Comment:
Where should the planner profile come from? Right now I'm extracting it from the request instructions composite it seems like that should be where the composite profile comes from